gradle/test-setup.gradle

```diff
@@ -18,22 +18,30 @@ tasks.withType(Test) { task ->
         events 'skipped'
     }
 
-    def dockerServerUrl = System.properties['dockerServerUrl']
-
-    if(dockerServerUrl) {
-        logger.quiet "Using Docker server URL '$dockerServerUrl' for executing tests with task '$task.name'."
+    if (project.hasProperty('bootstrapDocker') && Boolean.parseBoolean(project.getProperty('bootstrapDocker'))) {
```

Comment:
I would extract `if` logic into a method called `usingVagrant()` that would be more readable